addons/im_livechat/static/src/core/web/channel_invitation_patch.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="discuss.ChannelInvitation.main" t-inherit-mode="extension">
+        <xpath expr="//*[@name='selectablePartnerDetail']//*" position="inside">
+            <span t-if="props.thread?.channel_type === 'livechat' and selectablePartner.user_livechat_username" class="overflow-hidden opacity-75 smaller mx-2">
```

Comment:
Probably not necessary to play with opacity. We have `text-muted` available if necessary, but it might be fine in standard text.